internal/mikrotik/record.go

```diff
@@ -60,9 +60,14 @@ func NewRecordFromEndpoint(endpoint *endpoint.Endpoint) (*DNSRecord, error) {
 func NewEndpointFromRecord(record DNSRecord) (*endpoint.Endpoint, error) {
 	log.Debugf("converting Mikrotik DNS record: %v", record)
 
+	recType := record.Type
+	if recType == "" {
+		recType = "A"
+	}
```

Comment:
I like setting the record type like this since it's more verbose and it's better than a comment.

However, I'd say we should remove the empty string from the case statement that's a bit lower down in the file since it becomes redundant.

Reply:
Done, I think.